### DISCUSSION TITLE
Support for Aria by Rhymes AI

### DISCUSSION THREAD

**Discussion by daaain** [ggml-org/llama.cpp#9845] [Ideas]
If even half of the [announcement blog post](https://rhymes.ai/blog-details/aria-first-open-multimodal-native-moe-model) is true 😅 it's a really interesting model.

How difficult would it be to support it? I don't think there's any support for video yet?

**Comment by nkeilar** [comment]
Ollama is waiting for support here before it could be used in ollama/openwebui etc
